umbraco/Umbraco.CMS.Backoffice
# Feature: Data Type / Property Editor UI picker modal amends

## Description

Part feature, part bugfix, part chore... I've refactored the Data Type / Property Editor UI picker modals!

A few things...

- Aligned the property-editor UI groups aliases (in manifests) with v13, (we had a lot under "common" for v14)
- Deprecated `submitLabel` property (on `UmbDataTypePickerFlowModalData` and `UmbPropertyEditorUIPickerModalData` interfaces)
- Refactored/fixed the grouping code, to sort items alphabetically, and friendly group labels (`fromCamelCase`)
- Fixed the grouping code when filtering data-types, (performs a lookup against the property-editor UI data)

## Types of changes


## How to test?

- In the Document Type workspace > "Add property" > "Select property editor"...
  - Notice the property-editor UI grouping labels
  - Try filtering, e.g. "text", notice the "Available configurations" are now grouped
- Data Type workspace > "Select a property editor"... notice the property-editor UI grouping labels
